util/util.go

```diff
@@ -149,3 +150,26 @@ func SetSubtract(a []string, b []string) []string {
 	}
 	return diff
 }
+
+// HasEval determine whether matcher contains function eval
+func HasEval(s string) (bool, error) {
+	return regexp.MatchString(`eval\((.*)\)`, s)
```

Comment:
I think It's not very safe to just use `eval`, maybe we can change to `\beval` so that it wont match things like `xeval`

Reply:
I included `\b` in https://github.com/casbin/casbin-rs/pull/123 but not https://github.com/casbin/casbin-rs/pull/121, sorry

Reply:
Thanks for your advice, I've added it